service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -480,6 +482,12 @@ private HearingBooking buildFollowingHearings(CaseData caseData) {
 
         updateDocumentsBundleName(caseData, cancelledHearing);
 
+        if (caseData.getHearingOrdersBundlesDrafts() != null) {
+            updateHearingOrderBundles(caseData, cancelledHearing);
+        } else if (caseData.getDraftUploadedCMOs() != null) {
+            updateDraftUploadedCaseManagementOrders(caseData, cancelledHearing);
+        }
```

Comment:
Did you say there was a scenario where draft CMO can appear in both `draftUploadedCMOs` and `hearingOrdersBundlesDrafts`? If so we'll need to change this to update in both places.

Reply:
there may be draft CMOs in `draftUploadedCMOs` field. so I removed `else` so that both `draftUploadedCMOs` and `hearingOrdersBundlesDrafts` will be updated